pints/_mcmc/_nuts.py

```diff
@@ -0,0 +1,557 @@
+#
+# No-U-Turn Sampler (NUTS) MCMC method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import asyncio
+import pints
+import numpy as np
+
+
+class NutsState:
+    """
+    Class to hold information about the current state of the NUTS hamiltonian
```

Comment:
Could have `.` at the end of every sentence?

Reply:
done